tests/helpers/k8s.bash

```diff
@@ -628,3 +629,166 @@ function kindbox_cluster_teardown() {
   # Delete cluster config.
   rm -rf /root/.kube/${cluster}-config
 }
+
```

Comment:
Should we move these to a dedicated file (e.g., k3s.bash) given that k3s is not k8s ?

The k3s.bash file can always source the k8s.bash file to include its functions (e.g., k8s_node_ready, etc).

Reply:
Done.